dynamodb_test.go

```diff
@@ -72,14 +71,14 @@ func TestIntegrationWithCustomImageVersion(t *testing.T) {
 func TestIntegrationWithInvalidCustomImageVersion(t *testing.T) {
 	ctx := context.Background()
 
-	_, err := RunContainer(ctx, testcontainers.WithImage("amazon/dynamodb-local:0.0.7"))
+	_, err := Run(ctx, "amazon/dynamodb-local:0.0.7")
```

Comment:
...and refactor the invalid version next to the valid one, to make it more obvious.

Reply:
SGTM, thanks!